StantUiIosLibrary/Classes/Cells/CellWithIconAndDescriptionCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CellWithIconAndDescriptionCell.swift
+//  Pods-StantUiIosLibraryDemo
+//
+//  Created by Alexandre on 01/07/19.
+//
+
+import UIKit
+
+public class CellWithIconAndDescriptionCell: UITableViewCell {
+    
+    public static let IDENTIFIER = "CellWithIconAndDescriptionCellIdentifier"
+    public static var HEIGHT: CGFloat = 56
```

Comment:
Is this height changing somewhere? Put a "let" if it doesn't.

Reply:
I thought that somewhere that could change (someone that would use it could change the height), but I will change for let.